modules/DOM-action.js

```diff
@@ -0,0 +1,64 @@
+import BookClass from './book-class.js';
+// === declaration of variables === //
+let elBookContainer;
+let elID;
+let elTitle;
+let elAuthor;
+let elRemoveBtn;
+const elBookList = document.querySelector('.book-list');
+
+function toggoleRowColor() {
+  const bookContainers = document.querySelectorAll('.book-container');
+  for (let i = 0; i < bookContainers.length; i += 1) {
+    if (i % 2 === 0) bookContainers[i].style.backgroundColor = 'lightgray';
+  }
+}
+function fromJson(b) {
+  return new BookClass(b.id, b.title, b.author);
+}
+function removeAllChildNodes(parent) {
+  while (parent.firstChild) {
+    parent.removeChild(parent.firstChild);
+  }
+}
+
+export function createElements() {
```

Comment:
I like your clean code and ease of readability
- [ ] Let's kindly make sure we use ES6 Syntax (arrow functions) instead of regular function

I will remind you in next steps 😇

Reply:
@Amrhub "I like your clean code and ease of readability" 
Thank you 😃 

I will do all the functions as you said 😆